install/jakartaee/bin/sig-test.map

```diff
@@ -26,16 +26,12 @@
 #
 # The resultant signature file name has a name that complies
 # with the following format:
-# <package>.sig_<technology_version><JavaSE_version_these_work_with>
+# <package>.sig_<technology_version>
```

Comment:
Should the `<technology_version>` also be remove such that we end up signatures like `jakarta.annotation.sig`?

https://github.com/scottmarlow/jakartaee-tck/commit/9b7cd83b718dcbf3d565811468a674533574e5c5 shows what I mean (your pull request is more complete but you didn't seem to remove the api version).

Reply:
Corrected this comment and others.